cli_tools/gce_vm_image_import/importer/api_inflater.go

```diff
@@ -43,19 +43,24 @@ func isCausedByAlphaAPIAccess(err error) bool {
 
 // apiInflater implements `importer.inflater` using the Compute Engine API
 type apiInflater struct {
-	serialLogs    []string
-	args          ImportArguments
-	computeClient daisyCompute.Client
-	storageClient storage.Client
+	serialLogs      []string
+	args            ImportArguments
+	computeClient   daisyCompute.Client
+	storageClient   storage.Client
+	guestOsFeatures []*computeAlpha.GuestOsFeature
 }
 
 func createAPIInflater(args ImportArguments, computeClient daisyCompute.Client, storageClient storage.Client) inflater {
-	return &apiInflater{
+	inflater := apiInflater{
 		serialLogs:    []string{},
 		args:          args,
 		computeClient: computeClient,
 		storageClient: storageClient,
 	}
+	if args.UefiCompatible {
+		inflater.guestOsFeatures = []*computeAlpha.GuestOsFeature{{Type: "UEFI_COMPATIBLE"}}
```

Comment:
Not related to this bug, but are we setting "WINDOWS" guest OS feature in the API inflater? Maybe we should have a single func that adds these features (WINDOWS and UEFI_COMPATIBLE) in both API and regular inflater and just call it from both.

Reply:
I like the idea of consolidating these modifications. Tactically, based on the current implementation, I *think* we'll need to wait until the api_inflater is using the GA GCE client.  Or did you see an opportunity now?

Regarding the WINDOWS guest os feature, it seems to be required as well. @dntczdx  is this your reading too? Separate PR?

Reply:
Both inflaters modify disk Guest OS features based on args. That logic can be in a shared func.

Reply:
The structs have a different type though, right?
  - google.golang.org/api/compute/v1
  - google.golang.org/api/compute/v0.alpha

Reply:
Sure.

Reply:
Sounds good, and by then we will have a better position to do the test. So far I have no problem to do the uefi logic in this layer temporarily.